doc/Development_Documentation/05_Objects/03_Working_with_PHP_API.md

```diff
@@ -426,13 +426,13 @@ on `\Pimcore\Db\ZendCompatibility\QueryBuilder`.
  
 /** @var \Pimcore\Model\DataObject\Listing\Dao|\Pimcore\Model\DataObject\News\Listing $list */
 $list = new Pimcore\Model\DataObject\News\Listing();
- 
+$this->classId = $list->getClassId();
 // set onCreateQuery callback
 $list->onCreateQuery(
     function (\Pimcore\Db\ZendCompatibility\QueryBuilder $select) {
```

Comment:
function (\Pimcore\Db\ZendCompatibility\QueryBuilder $select) use ($list)

should work

Reply:
Ya this also😁